addons/ekco/template/base/install.sh

```diff
@@ -464,3 +466,14 @@ function ekco_load_images() {
         find "$DIR/image-overrides" -type f | xargs -I {} bash -c "cat {} | ctr -a $(${K8S_DISTRO}_get_containerd_sock) -n=k8s.io images import -"
     fi
 }
+
+function ekco_dynamicstorage() {
+    if [ "$ROOK_MINIMUM_NODE_COUNT" -gt "1" ]; then
```

Comment:
what if ROOK_MINIMUM_NODE_COUNT is not set? will it be empty? in this case dont you need to check that there is a value?

```suggestion
    if [ -n "$ROOK_MINIMUM_NODE_COUNT" ] && [ "$ROOK_MINIMUM_NODE_COUNT" -gt "1" ]; then
```